package-lock.json

```diff
@@ -1,70 +1,248 @@
 {
   "name": "azure-devops-extension-sdk",
-  "version": "2.0.11",
+  "version": "3.1.10",
```

Comment:
Version doesn't match here. Looks like the version was changed in pcakge.json after packge-lock.json was created

Reply:
Done. Thanks